selfdrive/test/test_car_models.py

```diff
@@ -441,6 +441,14 @@ def get_route_log(route_name):
     'carFingerprint': VOLKSWAGEN.GOLF,
     'enableCamera': True,
   },
+  "58a7d3b707987d65|2021-03-25--17-26-37": {
+    'carFingerprint': VOLKSWAGEN.JETTA_MK7,
+    'enableCamera': True,
+  },
+  "ea70677b54a1bca1|2020-06-02--16-55-55": {
```

Comment:
this one's gone

Reply:
Pulled Passat till I can get a route from one of the two owners tomorrow AM, I'll put it in a separate PR then.